### PR TITLE
Add tc-web FE to monorepo

### PR DESCRIPTION
**Pull request by owlishDeveloper**
I don't quite understand how to go about `.taskcluster.yml` - add this dir to the root one, just like other services?

Also, linting. Does it get to have its own linting rules?

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 services/queue @jhford
 services/web-server @taskcluster/frontend-reviewers
+taskcluster-web @taskcluster/frontend-reviewers
```
Comment (djmitche) [inline .github/CODEOWNERS]: You're going to hate me, sorry, but I kind of want to just call it `web/` or something else short and sweet (`frontend`? `ui`?).  We've eliminated `taskcluster-` from all of the other pathnames in this repo.  Hopefully once a name is selected this isn't a hard fix..

Comment (owlishDeveloper) [reply .github/CODEOWNERS]: Oh, let's call it `ui`, PLEASE!

**Comment by owlishDeveloper** [comment]
@helfi92 I wonder if we should try deploying it on Netlify? Before merging. Are you able to start this thing locally?

File: netlify.toml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+[build]
```
Comment (djmitche) [inline netlify.toml]: It's probably worth a header comment here explaining how we use Netlify (and that it's not used in production)

File: test/meta_test.js [hunk 3]
```diff
@@ -34,6 +36,10 @@ suite('Repo Meta Tests', function() {
     assert.equal(taskclusterYml.tasks.$let.node, packageJson.engines.node);
   });
 
+  test('Node version for UI matches the rest of the repo', function() {
+    assert.equal(taskclusterYml.tasks.$let.node, uiPackageJson.engines.node);
+  });
+
```
Comment (djmitche) [inline test/meta_test.js]: :tada:

**Comment by helfi92** [comment]
Please refrain from merging today. I would like to review it tomorrow morning.

**Comment by djmitche** [comment]
> sheep it!

that's my opinion only - sorry if it sounded too eager :)

**Comment by helfi92** [comment]
No worries :)

File: netlify.toml (hunk 2 above)
Comment (owlishDeveloper) [reply netlify.toml]: I think once it goes to production there'll be no Netlify anyways?

Comment (djmitche) [reply netlify.toml]: I think we'll still use netlify to test PRs?

**Pull request merged**